lib/lib_rss.php

```diff
@@ -189,7 +189,14 @@ function html_only_entity_decode($text) {
 }
 
 function prepareSyslog() {
-	return COPY_SYSLOG_TO_STDERR ? openlog("FreshRSS", LOG_PERROR | LOG_PID, LOG_USER) : false;
+	if (defined('SYSLOG_PREPARED')) {
+		return null;
+	}
+	define('SYSLOG_PREPARED', true);
+	return COPY_SYSLOG_TO_STDERR ? openlog('FreshRSS', LOG_PERROR | LOG_PID, LOG_USER) : false;
```

Comment:
I'm not sure what's the purpose of this environment var, shouldn't we use `COPY_LOG_TO_SYSLOG` instead? (and drop this one)

Reply:
It is for different cases. FreshRSS has normal text logs, and syslogs (e.g. for SimplePie, favicons...).

* `COPY_LOG_TO_SYSLOG`: Depending on whether you are monitoring FreshRSS text logs or not, this might be relevant, or in the contrary create annoying duplicates.

* `COPY_SYSLOG_TO_STDERR`: In a Docker deployment for instance, copying to stderr is convenient so that those lines are visible in Docker logs, but for other systems with a native syslog (like when FreshRSS is installed natively), then we should not do it.

In our Docker image (where there is no native syslog), we use both, but any combination can be relevant in my opinion.

Reply:
Ok, right. But we should at least call `openlog` with `COPY_LOG_TO_SYSLOG`: on my server there's a strange behavior where FRSS logs are identified as another app that called `openlog` before.

Reply:
Ah, really? Can you explain a bit more the situation?

Reply:
Ok, should be better with https://github.com/FreshRSS/FreshRSS/pull/2745/commits/48ab4ee05fe66989ff5c5d0ad878db4319d5bcb1

Reply:
Well, I had `COPY_LOG_TO_SYSLOG=On` and `COPY_SYSLOG_TO_STDERR=Off`, so the `openlog` wasn't called in FRSS. And for some reasons, the `syslog` calls were considering the `ident` string set by another application, so my logs were all mixed up :upside_down_face:

Reply:
Ok, strange, but is it better now? :-)

Reply:
It looks good, yes :)